src/leader.py

```diff
@@ -152,6 +155,38 @@ def lead(self):
             self.turn(code, position)
 
 
+    def drive_in_circle(self):
+        """Drives in a circle."""
+        self.bw.speed(MAX_SPEED)
+        self.bw.forward()
+
+        self.turn_left()
+
+
+    def drive_wide_winding(self):
+        """Drives in a wide, snake-like pattern."""
+        self.bw.speed(MAX_SPEED)
+        self.bw.forward()
+
+        self.turn_right()
+        sleep(DEMO_TURN_TIME)
```

Comment:
I don't think this works. If you call sleep, then it blocks the whole leader script from doing anything for a whole second until it resumes again to turn right and sleeps some more.